modules/my_health/spec/request/v1/messages_request_spec.rb

```diff
@@ -9,327 +9,377 @@
-        expect(first_message['triage_group_name']).to eq('TG API TESTING')
-        expect(first_message['has_attachments']).to eq(false)
-        expect(first_message['subject']).to eq('Test Inquiry')
-        expect(first_message['category']).to eq('TEST_RESULTS')
-        expect(first_message['folder_id']).to eq(-2)
-        expect(first_message['message_body']).to eq('TEST0101010101')
-        expect(first_message['proxy_sender_name']).to be_nil
-        expect(first_message['read_receipt']).to be_nil
-        expect(response).to be_successful
-      end
+        it 'responds to DELETE' do
+          VCR.use_cassette('sm_client/messages/deletes_the_message_with_id') do
+            delete "/my_health/v1/messaging/messages/#{message_id}"
+          end
 
-      it 'responds to GET #thread with full_body query param' do
-        VCR.use_cassette('sm_client/messages/gets_a_message_thread_full_body') do
-          get "/my_health/v1/messaging/messages/#{thread_id}/thread?full_body=true"
+          expect(response).to be_successful
+          expect(response).to have_http_status(:no_content)
         end
+      end
 
-        json_response = JSON.parse(response.body)
-        data = json_response['data']
-
-        expect(data).to be_an(Array)
-
-        first_message = data.first['attributes']
-        expect(first_message['message_id']).to eq(3_207_476)
-        expect(first_message['attachments']).to be_empty
-
-        second_message = data[1]['attributes']
-        expect(second_message['message_id']).to eq(3_204_755)
-
-        attachments = second_message['attachments']
-        expect(attachments.length).to eq(2)
+      describe '#move' do
+        let(:message_id) { 573_052 }
 
-        first_attachment = attachments.first
-        expect(first_attachment['id']).to eq(3_204_753)
-        expect(first_attachment['name']).to eq('almost4mbfile.pdf')
-        expect(first_attachment['attachment_size']).to eq(3_976_877)
-        expect(first_attachment['message_id']).to eq(3_204_755)
+        it 'responds to PATCH messages/move' do
+          VCR.use_cassette('sm_client/messages/moves_a_message_with_id') do
+            patch "/my_health/v1/messaging/messages/#{message_id}/move?folder_id=0"
+          end
 
-        third_message = data[2]['attributes']
-        expect(third_message['message_id']).to eq(3_203_739)
-        expect(third_message['message_body'].length).to be > 200
+          expect(response).to be_successful
+          expect(response).to have_http_status(:no_content)
+        end
       end
+    end
+  end
 
-      it 'responds to GET #thread when camel-inflected' do
-        VCR.use_cassette('sm_client/messages/gets_a_message_thread_full') do
-          get "/my_health/v1/messaging/messages/#{thread_id}/thread", headers: { 'X-Key-Inflection': 'camel' }
-        end
+  context 'when legacy sm policy' do
+    before do
+      Flipper.disable(:mhv_sm_session_policy)
```

Comment:
same here, is you allow(Flipper)... you won't have the possibility of running into flaky specs